templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -0,0 +1,146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine-tuning Llama-2/Mistral models with Anyscale\n",
```

Comment:
Please add estimated time to completion, see other templates for example.

Reply:
Updated. Added different time based on models